backend/app/services/collections/providers/base.py

```
@@ -1,9 +1,8 @@
 from abc import ABC, abstractmethod
-from typing import Any
+from typing import Any, List
```

Ruff (0.15.6)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)